Integrations/integration-Netskope.yml

```diff
@@ -0,0 +1,332 @@
+configuration:
+- display: URL of Netskope Tenant (e.g. tenant.goskope.com)
+  name: url
+  defaultvalue: ""
+  type: 0
+  required: true
+- display: Tenant API Token
+  name: token
+  defaultvalue: ""
+  type: 0
+  required: true
+- display: Do not validate server certificate (insecure)
+  name: insecure
+  defaultvalue: "false"
+  type: 8
+  required: false
+- display: Use system proxy settings
+  name: proxy
+  defaultvalue: "false"
+  type: 8
+  required: false
+script:
+  script: |-
+    var url = params.url.replace(/[\/]+$/, '');
+    var token = params.token;
+    var context = [];
+
+    if (url[url.length - 1] !== '/') {
+        url += '/';
+    }
+
+    function sendRequest(method, url) {
+        var result = http(
+            url,
+            {
+                Headers: {'Content-Type': ['application/json'], 'Accept': ['application/json']},
+                Method: method
+            },
+            params.insecure,
+            params.proxy
+        );
+
+        if (result.StatusCode < 200 && result.StatusCode > 299) {
+            throw 'Failed to perform request ' + path + ', request status code: ' + result.StatusCode + ', body: ' + result.Body;
+        }
+        if (result.Body === '') {
+            throw 'No content received.' + url + result;
+        }
+        var obj;
+        try {
+            obj = JSON.parse(result.Body);
+        } catch (ex) {
+            throw 'Error parsing reply - ' + result.Body + ' - ' + ex;
+        }
+        return {
+            body: result.Body,
+            obj: obj,
+            statusCode: result.StatusCode
+        };
+    }
+
+    function isEmpty(object) {
+        if(Array.isArray(object)){
+            return object.length === 0;
+        }
+        for(var i in object) {
+            if(object[i]){
+                return false;
+            }
+        }
+        return true;
+    }
+
+    function splitToID(str){
+        var temp = str.split('/');
+        return temp[temp.length -1];
+    }
+
+    function removeFromArray(keys, array){
+        for(var i=0; i<keys.length; i++){
+            array.splice(array.indexOf(keys[i]), 1);
+        }
+        return array;
+    }
+
+    function netskopeEvents(result) {
+        var res = JSON.parse(result.Data);
+
+        var md;
+    }
+
+    switch (command) {
+        case 'test-module':
+            if (sendRequest('GET', "https://" + url + "/api/v1/events?token=" + token + "&type=application&timeperiod=3600&limit=1")) {
```

Comment:
Even when the token is invalid the test will return ok. But after checking the what sendRequest returns is:
![image](https://user-images.githubusercontent.com/7270217/38448531-6f3dc61a-3a0d-11e8-9d50-05eb924f6866.png)